src/app/genes/gene-overview/soe/soe-chart/soe-chart.component.scss

```diff
@@ -6,6 +6,7 @@
   flex-direction: row;
   flex-wrap: wrap;
   justify-content: space-between;
+  gap: 20px;
```

Comment:
Trying to fix the arrangement of the graphs with this. When I add a gap, the layout seems more consistent. Can't do much more unless we want to use something other than flex

Reply:
ok, I hope it works!